### PR TITLE
POL-101 Azure Instance Rightsizing

### PR DESCRIPTION
**Pull request by srpomeroy**
### Description

This policy will rightsize Azure instances based on utilization metrics from Log Analytics.

### Issues Resolved

N/A

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/azure/instances_log_analitics_rightsizing/azure_instance_log_analytics_rightsizing.pt [hunk 1]
```diff
@@ -0,0 +1,777 @@
+name "Azure Instances Rightsizing from Log Analytics"
```
Comment (cdwilhelm) [inline cost/azure/instances_log_analitics_rightsizing/azure_instance_log_analytics_rightsizing.pt]: Change to: Azure Compute Instance Rightsizing

File: cost/azure/instances_log_analitics_rightsizing/azure_instance_log_analytics_rightsizing.pt [hunk 2]
```diff
@@ -0,0 +1,777 @@
...
+rs_pt_ver 20180301
+type "policy"
+short_description "Rightsizes Azure instance based on utilization data from Azure Log Analytics.  See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/instances_log_analitics_utilization) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```
Comment (cdwilhelm) [inline cost/azure/instances_log_analitics_rightsizing/azure_instance_log_analytics_rightsizing.pt]: Change to: Rightsize Azure Compute Instances based on CPU and Memory threshold

File: cost/azure/instances_log_analitics_rightsizing/azure_instance_log_analytics_rightsizing.pt [hunk 3]
```diff
@@ -0,0 +1,777 @@
...
+  cooldown_instances: cooldown_instances
+}
+EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy 'policy_azure_utilization' do
```
[the remark this answers is not in the document]
Comment (srpomeroy) [reply cost/azure/instances_log_analitics_rightsizing/azure_instance_log_analytics_rightsizing.pt]: Nope. Removed.

File: cost/azure/instance_rightsizing/azure_instance_rightsizing.pt [hunk 4]
```diff
@@ -0,0 +1,732 @@
+name "Azure Rightsize Virtual Machine Instances"
+rs_pt_ver 20180301
+type "policy"
+short_description "Rightsizes Azure virtual machine instances based on CPU and Memory thresholds.  See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/instance_rightsizing) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```
Comment (cdwilhelm) [inline cost/azure/instance_rightsizing/azure_instance_rightsizing.pt]: use:  Finds Azure Virtual Machine instances based on CPU and Memory thresholds and resizes them after approval.

File: cost/azure/instance_rightsizing/README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+# Azure Instance Rightsizing
```
Comment (cdwilhelm) [inline cost/azure/instance_rightsizing/README.md]: Change to match policy name

**Pull request closed without merging**